content/en/tracing/other_telemetry/connect_logs_and_traces/php.md

```diff
@@ -67,6 +67,21 @@ If the logger implements the [**monolog/monolog** library][4], use `Logger::push
 ?>
 ```
 
+For monolog v2:
+
+```php
+<?php
+
+  $logger->pushProcessor(function ($record) {
+      $context = \DDTrace\current_context();
+      return $record->with(message: $record['message'] . sprintf(
+          ' [dd.trace_id=%s dd.span_id=%s]',
+          $context['trace_id'],
+          $context['span_id']
+      ));
+  });
+```
```

Comment:
I think we should try to fit it stylistically in line with the other examples, i.e. without extra newline after `<?php` and a closing `?>`.

Reply:
updated @bwoebi. Look ok?

Reply:
Yep :-)